internal_deps.bzl

```diff
@@ -20,20 +20,20 @@ def rules_webpack_internal_deps():
 
     http_archive(
         name = "io_bazel_rules_go",
-        sha256 = "099a9fb96a376ccbbb7d291ed4ecbdfd42f6bc822ab77ae6f1b5cb9e914e94fa",
-        urls = ["https://github.com/bazelbuild/rules_go/releases/download/v0.35.0/rules_go-v0.35.0.zip"],
+        sha256 = "80a98277ad1311dacd837f9b16db62887702e9f1d1c4c9f796d0121a46c8e184",
```

Comment:
blah! we should remove go from this repo, we have no Go sources, and `aspect configure` is a statically-linked binary

Reply:
follow-up